### ISSUE TITLE
[A11y] Desktop/Mobile Explore Collection -Grid: Tab Order of Image Thumbnails

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1799]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Explore Collections

### Admin user?

- [ ] Yes

### Issue Description

Explore Collection Grid - https://staging.oregondigital.org/explore_collections/all?locale=en&view=masonry

The thumbnail tab order at the bottom is not consecutive. See thumbnail label #21-#24 showing the tab order

<img width="1341" alt="Screen Shot 2021-08-04 at 7 51 53 AM" src="https://user-images.githubusercontent.com/8353860/128203109-2226ff1c-7ea4-4ef5-926c-e2d2d56dce31.png">

### Suggested Fix

Instead of 21, 23 , 22, 24 Should be 21, 22, 23, 24

**Comment by CGillen** [comment]
While this looks like a grid view where tab order should logically flow left to right, this is actually a masonry where tab order is a little less obvious but still top to bottom, left to right.
It looks weird at first glance, but the example isn't a failure
